<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addScientistWindow</class>
 <widget class="QDialog" name="addScientistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_addsci_header">
     <property name="text">
      <string>&lt;h3&gt;Add a new scientist&lt;/h3&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_addsci_name">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_addsci_name">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Enter name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_addsci_died">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Year of death:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="input_addsci_born">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333;</string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="placeholderText">
        <string>Enter year</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="input_addsci_died">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="placeholderText">
        <string>Enter year</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_addsci_born">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Year of birth:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="input_addsci_gender"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_addsci_gender">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_addsci_status">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_addsci_add">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid #333;
padding: 7px 0 5px 0;
background-color: #5EC748;
color: #fff;
font: 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>Add Scientist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_addsci_done">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid #333;
padding: 7px 0 5px 0;
background-color: #555;
color: #fff;
font: 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
